core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/typefunction/ToNumberExpression.kt

```diff
@@ -21,9 +22,20 @@ class ToNumberExpression<T : ValidType>(
     }
 }
 
-fun <T : ValidType> toNumber(expression: TypeExpression<T>) = ToNumberExpression(expression)
+fun <T : ValidType> TypeExpression<T>.toNumber() = ToNumberExpression(this)
 
-fun toNumber(
-    expression: TypeExpression<StringType>,
-    stringExpression: TypeExpression<StringType>,
-) = ToNumberExpression(expression, stringExpression)
+fun String.toNumber() = ToNumberExpression(this.toDopeType())
+
+fun Boolean.toNumber() = ToNumberExpression(this.toDopeType())
+
+fun TypeExpression<StringType>.toNumber(stringExpression: TypeExpression<StringType>) =
```

Comment:
can we name the parameter to something more meanigful? Like _filterChars_ or _removeChars_?

Reply:
I like _filterChars_ 🙂 I renamed it